internal/app/connectconformance/server_runner.go

```diff
@@ -236,8 +237,9 @@ func runTestCasesForServer(
 			wg.Done() // call it explicitly since callback above won't be invoked
 			// client pipe broken: mark remaining tests, including this one, as failed
 			for j := i; j < len(testCases); j++ {
-				results.setOutcome(testCases[j].Request.TestName, true, err)
+				results.setOutcome(testCases[j].Request.TestName, true, &errCouldNotRun{err})
 			}
+			break
```

Comment:
This is the reason that, even after the client stopped, the `--vv` output would continue to show lots of "Sending request for ..." messages, because after detecting the error we didn't actually break out of this loop. Oops.